modules/environment/login/nix-on-droid.nix.default

```diff
@@ -28,23 +28,25 @@
   environment.etcBackupExtension = ".bak";
 
   # Read the changelog before changing this value
-  system.stateVersion = "21.05";
+  system.stateVersion = "21.11";
 
   # After installing home-manager channel like
-  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-21.05.tar.gz home-manager
+  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-21.11.tar.gz home-manager
   #   nix-channel --update
   # you can configure home-manager in here like
   #home-manager.config =
   #  { pkgs, lib, ... }:
   #  {
   #    # Read the changelog before changing this value
-  #    home.stateVersion = "21.05";
+  #    home.stateVersion = "21.11";
   #
   #    # Use the same overlays as the system packages
-  #    nixpkgs.overlays = lib.mkIf (lib.hasAttrByPath [ "nixpkgs" "overlays" ] config) config.nixpkgs.overlays;
+  #    nixpkgs = { inherit (config.nixpkgs) overlays; };
```

Comment:
Is the #134 case covered enough by the on-device flake test?

Reply:
Now I finally understood the whole issue and why this fix was needed. I need to do some changes in `modules/nixpkgs.nix` to let this one work. (When in a flake setup, you should not be allowed to change the `pkgs` instance, but the option should be available to be at least able to evaluate the config.)

To your question: No, we should have a test case for home-manager integration for flake and non-flake setup. To reproduce the issue of #134, you should set `nixpkgs.overlays = [ ];` in a flake setup, I think.

Reply:
Not sure if 8511636 has resolved it, getting the same #134 symptoms...